README.md

```diff
@@ -1,18 +1,20 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Leaderboard
 
-> One paragraph statement about the project.
+> Leaderboard to compare users scores!
 
-![screenshot](./app_screenshot.png)
+![screenshot](./printscreen.png)
```

Comment:
- [ ] The link in README is broken. Kindly fix it to make README more professional.

Reply:
I tried to 3 times, do you have any idea of what might be breaking it??

Reply:
the image file name is `printscreen.PNG`, the PNG is in capital. try making it in lowercase and check.

Reply:
It worked, thanks.